component/common/pom.xml

```diff
@@ -78,6 +78,10 @@
             <groupId>org.powermock</groupId>
             <artifactId>powermock-module-testng</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Lets not add this with testing dependencies. Add this with logging dependencies.